tuta-sdk/rust/src/user_facade.rs

```diff
@@ -0,0 +1,31 @@
+use std::borrow::ToOwned;
+use std::sync::Arc;
+use crate::{ApiCallError, TypeRef};
+use crate::entities::sys::User;
+use crate::entity_client::IdType;
+use crate::typed_entity_client::TypedEntityClient;
+
+
+/// Provides high level functions to manipulate user entities via the REST API
```

Comment:
The comment is wrong.

Currently in webapp `UserFacade` does not load anything and just holds the state. I am not sure if you intended it in the same way?

Reply:
I changed it to a `FIXME`.